Core/EventHandlers.lua

```diff
@@ -1104,9 +1109,9 @@ function R:OnBagUpdate()
 	R:ProcessInventoryItems()
 end
 
---[[
-      OBTAIN DETECTION ---------------------------------------------------------------------------------------------------------
-      -- Some easy, some fairly arcane methods to detect when we've obtained something we're looking for
+--[[
+      OBTAIN DETECTION ---------------------------------------------------------------------------------------------------------
+      -- Some easy, some fairly arcane methods to detect when we've obtained something we're looking for
```

Comment:
guess so yeah

Reply:
going to line 1107 in eventhandlers.lua and saving (to trigger format) does nothing to that comment block but still triggers a change in the file on github desktop, not too sure whats happening there

Reply:
It seems a carriage return was added when you saved the file. It's probably because you haven't turned off `git config core.autocrlf`  - see [this StackOverflow thread](https://stackoverflow.com/questions/2825428/why-should-i-use-core-autocrlf-true-in-git). If you set that to `false`, then undo/redo the changes, it should be gone.